codestodians/food-basket-api
# [Feat: #154589946] Add Forgot password Mutation

#### What does this PR do?
- Adds a mutation to enable a link to be sent to the user when the password is forgotten.

#### Description of Task to be completed?
- A user should be able to reset his/her password if forgotten.

#### How should this be manually tested?
- create a sendgrid account and generate an API Key
- check out into the `feat/forgot-password-154589946` branch
- add the `SENDGRID_API_KEY` to .env
- run `pip3 install -r requirements.txt`to install newly added packages
- start the server with `gunicorn config.wsgi`
- run the `forgotPassword` mutation on `GRAPHIQL`, supplying `email` as an argument to the mutation.
- The email supplied should receive a mail regarding password reset.

#### Any background context you want to provide?
- The actual link to be sent to the user was not added in this story as we are yet to implement a valid url to change password.

#### What are the relevant pivotal tracker stories?
[#154589946](https://www.pivotaltracker.com/story/show/154589946)
